.travis.yml

```diff
@@ -24,30 +35,67 @@ matrix:
   include:
     - os: linux
       env: MB_PYTHON_VERSION=2.7
+    - os: linux
+      env:
+        - MB_PYTHON_VERSION=2.7
+        - UNICODE_WIDTH=16
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.5
+        - NP_BUILD_DEP=numpy==1.9.3
+        - NP_TEST_DEP=numpy==1.9.3
+    - os: linux
+      env:
+        - MB_PYTHON_VERSION=3.6
+        - NP_BUILD_DEP=numpy==1.11.3
+        - NP_TEST_DEP=numpy==1.11.3
     - os: osx
-      osx_image: xcode6.4
       language: generic
       env:
         - MB_PYTHON_VERSION=2.7
     - os: osx
-      osx_image: xcode6.4
       language: generic
       env:
         - MB_PYTHON_VERSION=3.5
+        - NP_BUILD_DEP=numpy==1.9.3
+        - NP_TEST_DEP=numpy==1.9.3
+    - os: osx
+      language: generic
+      env:
+        - MB_PYTHON_VERSION=3.6
+        - NP_BUILD_DEP=numpy==1.11.3
+        - NP_TEST_DEP=numpy==1.11.3
+
 before_install:
+    - if [ "$TRAVIS_BRANCH" == "master" ]; then
```

Comment:
What is going on here?

Reply:
It's probably the wrong way round, but the idea is, if you're building on "master", the wheels should go up to Rackspace attached to a "release" URL [here](https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com/), whereas if you are building on - say - a "daily" branch, the wheels to a pre-release container [here](https://7933911d6844c6c53a7d-47bd50c35cd79bd838daf386af554a83.ssl.cf2.rackcdn.com).

I say the wrong way round, because it's probably better to use "master" for the regular test builds / pre-releases, and other branches for releases.